src/index.html

```diff
@@ -11,7 +11,175 @@
+          Full date of birth:
+          <input
+            type="date"
+            name="dob"
+          >
+        </label>
+
+        <label class="form-field ">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="agreement"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="registration groups">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Password:
+          <input
+            type="password"
+            name="password"
+            required
+            minlength="5"
+            maxlength="10"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fact groups">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="catLover"
+              value="Yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input
+              type="radio"
+              name="catLover"
+              value="No"
+            >
+            No
+          </label>
+        </label>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="time"
+          >
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select
+            name="carBrands"
+            multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          How do you rate our work?
+          <input
+            type="range"
+            name="rating"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="info groups">
+        <legend>Additional info:</legend>
+
+        <label class="form-field">
+          Comments:
+          <textarea
+            name="comment"
+            cols="20"
+            rows="2"
+          >
+          </textarea>
+        </label>
+
+        <label class="form-field">
+          Would you recommend us?
+          <select name="recommend">
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
+      </fieldset>
+
+      <button
+        type="submit"
+      >
```

Comment:
It's short enough to put in one line.
```suggestion
      <button type="submit">
```